test/stub-test.js

```diff
@@ -119,6 +119,94 @@ describe("stub", function () {
         });
     });
 
+    describe(".resolves", function () {
+        it("returns a promise to the specified value", function () {
+            var stub = createStub.create();
+            var object = {};
+            stub.resolves(object);
+
+            return stub().then(function (actual) {
+                assert.same(actual, object);
+            });
+        });
+
+        it("should return stub", function () {
+            var stub = createStub.create();
+
+            assert.same(stub.resolves(""), stub);
+        });
+
+        it("supersedes previous throws", function () {
+            var stub = createStub.create();
+            stub.throws().resolves(1);
+
+            refute.exception(function () {
+                stub();
+            });
+        });
+
+        it("supersedes previous rejects", function () {
+            var stub = createStub.create();
+            stub.rejects(Error("should be superseeded")).resolves(1);
+
+            return stub().then();
+        });
+    });
+
+    describe(".rejects", function () {
+        it("returns a promise which rejects for the specified reason", function () {
+            var stub = createStub.create();
+            var reason = new Error();
+            stub.rejects(reason);
+
+            return stub().then(function () {
+                referee.fail("this should not resolve");
+            }).catch(function (actual) {
+                assert.same(actual, reason);
+            });
+        });
+
+        it("returns stub", function () {
```

Comment:
Sorry, did not see this until I merged ... oh well, no biggie, although I like your suggestion :)